website/docs/reference/events.mdx

```diff
@@ -291,7 +291,7 @@ Generated when you drop all existing tags as part of a configuration import. The
 
 #### `feature-potentially-stale-on`
 
-Generated when Unleash marks a feature flag as potentially stale due to exceeding the expected lifetime of its [feature flag type](/reference/feature-toggle-types). `preData` and `data` are null.
+Generated when Unleash marks a feature flag as potentially stale due to exceeding the expected lifetime of its [feature flag type](/reference/feature-toggles#feature-flag-types). `preData` and `data` are null.
```

Comment:
```suggestion
Generated when Unleash marks a feature flag as potentially stale due to exceeding the expected lifetime of its [feature flag type](/reference/feature-toggles#feature-flag-types). `preData` and `data` are `null`.
```

Reply:
Why do you suggest code formatting for this? Given that it's an unrelated change, and there are many occurrences of it, let's tackle it separately